build.sbt

```diff
@@ -39,8 +37,7 @@ lazy val buildSettings = Seq(
         <email>dev-role@allenai.org</email>
       </developer>
     </developers>,
```

Comment:
JCenter might require some or all of this. We should figure out what's required.

If maven central requires this, it's likely we won't be able to use the replication from JCenter to maven central without it . . .